packages/expo-contacts/ios/ContactsRequester.swift

```diff
@@ -30,10 +30,6 @@ class ContactsPermissionRequester: NSObject, EXPermissionsRequester {
       guard let self else {
         return
       }
-      if let error {
-        reject("E_CONTACTS_ERROR_UNKNOWN", error.localizedDescription, error)
-        return
-      }
```

Comment:
If the error is not necessary, we can omit the parameter otherwise there will be a warning. The warnings from Xcode don't work correctly in bare expo.

Reply:
> If the error is not necessary, we can omit the parameter otherwise there will be a warning. The warnings from Xcode don't work correctly in bare expo.

Right, no Swift expert here admittedly, pushed a new commit that replaces `error` for `_`

Reply:
No problem 👍